plugins/com.google.cloud.tools.eclipse.appengine.facets.test/src/com/google/cloud/tools/eclipse/appengine/facets/AppEngineFlexFacetTest.java

```diff
@@ -0,0 +1,14 @@
+package com.google.cloud.tools.eclipse.appengine.facets;
+
+import org.eclipse.wst.common.project.facet.core.ProjectFacetsManager;
+import org.junit.Assert;
+import org.junit.Test;
+
+import com.google.cloud.tools.eclipse.appengine.facets.AppEngineFlexFacet;
+
+public class AppEngineFlexFacetTest {
+  @Test
+  public void testFlexFacetExists() {
+    Assert.assertTrue(ProjectFacetsManager.isProjectFacetDefined(AppEngineFlexFacet.ID));
```

Comment:
If you use the string literal here instead of the constant, then you also test the value of the constant.